readthedocs/doc_builder/config.py

```diff
@@ -108,6 +106,15 @@ def formats(self):
             formats += ['pdf']
         return formats
 
+    @property
+    def build_image(self):
+        if self._project.container_image:
+            # Allow us to override per-project still
+            assert 'readthedocs/build' in self._project.container_image, (
```

Comment:
How does this `assert` work?

Shouldn't we raise a proper exception like ConfigError or InvalidConfig here?

Reply:
Yea, I know I've set it wrong before in the admin, so like having clarity there. Can definitely raise other errors though :)